src/box/alter.cc

```diff
@@ -4325,6 +4325,8 @@ on_commit_schema_set_bootstrap_leader_uuid(struct trigger *trigger, void *event)
 	struct tt_uuid *uuid = (struct tt_uuid *)trigger->data;
 	bootstrap_leader_uuid = *uuid;
 	is_supervised_bootstrap_leader = tt_uuid_is_equal(uuid, &INSTANCE_UUID);
+	say_info("instance %s is assigned as a bootstrap leader",
```

Comment:
Let's add some details here:

You could print `This instance (uuid) is assigned as a bootstrap leader` judging by `is_supervised_bootstrap_leader`. Would be  bit more informative IMO and more similar to the other message from `make_bootstrap_leader` itself.

Reply:
@Serpentian suggested the opposite :) See https://github.com/tarantool/tarantool/pull/10859#discussion_r1861378859.

I don't mind any option, TBH.

Reply:
Oh, ok. Let's see what @Serpentian thinks.

Reply:
Nit: 
I'd print `This instance (uuid)`, not simply `This instance ...`

Reply:
https://github.com/tarantool/tarantool/pull/10859#discussion_r2023182732

Reply:
> [#10859 (comment)](https://github.com/tarantool/tarantool/pull/10859#discussion_r2023182732)

Haven't thought of this, that's a nice idea.